internal/pkg/v2/infrastructure/redis/device.go

```diff
@@ -0,0 +1,82 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package redis
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/edgexfoundry/edgex-go/internal/pkg/common"
+	"github.com/edgexfoundry/go-mod-core-contracts/errors"
+	model "github.com/edgexfoundry/go-mod-core-contracts/v2/models"
+
+	"github.com/gomodule/redigo/redis"
+)
+
+const DeviceCollection = "v2:device"
+
+// deviceStoredKey return the device's stored key which combines the collection name and object id
+func deviceStoredKey(id string) string {
+	return fmt.Sprintf("%s:%s", DeviceCollection, id)
+}
+
+// deviceExistByName whether the device exists by name
+func deviceExistByName(conn redis.Conn, name string) (bool, errors.EdgeX) {
```

Comment:
rename the naming convention to deviceNameExists, and modify others

Reply:
Modified.